index.d.ts

```diff
@@ -1361,7 +1361,7 @@ export interface SegmentedControlProps extends Omit<React.ComponentPropsWithoutR
   height?: number
 }
 
-export class SegmentedControl extends React.PureComponent<SegmentedControlProps> {
+export class SegmentedControl extends React.FC<SegmentedControlProps> {
```

Comment:
`export declare const SegmentedControl: ForwardRefComponent<>`?

Reply:
Oh I see there's no ref support, this should just be `export const SegmentedControl: React.FC<>` maybe?